bindings/azure/servicebusqueues/servicebusqueues.go

```diff
@@ -187,10 +175,19 @@ func (a *AzureServiceBusQueues) Invoke(ctx context.Context, req *bindings.Invoke
 		msg.TimeToLive = &ttl
 	}
 
-	ctx, cancel := context.WithTimeout(ctx, a.timeout)
-	defer cancel()
+	// Send the message
+	ctx, cancel := context.WithTimeout(invokeCtx, a.timeout)
+	err = sender.SendMessage(ctx, msg, nil)
+	cancel()
```

Comment:
Why no more defer on cancel()?

Reply:
Just my old habits of avoiding defers :) (defer used to be slow -  although it shouldn't be the case anymore).

Reply:
Changed